Bottom: a382695e2987cbd0d5a442f0e357413683b8c1df
Top:    dc40d7521b90802ab52de937f20882c5d9cf6403
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-16 14:26:41 -0700

Refresh of waitall-buggy

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 2edd44e..a7692c5 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1133,8 +1133,7 @@ ssize_t rrecv(int socket, void *buf, size_t len, int flags)
 	fastlock_acquire(&rs->rlock);
 	do {
 		if (!rs_have_rdata(rs)) {
-			ret = rs_get_comp(rs, rs_nonblocking(rs, flags),
-					  rs_conn_have_rdata);
+			ret = rs_get_comp(rs, rs_nonblocking(rs, flags), rs_conn_have_rdata);
 			if (ret)
 				break;
 		}
